package/native-package/package.json

```diff
@@ -14,17 +14,17 @@
     "stream-chat-react-native-core": "5.33.0"
   },
   "peerDependencies": {
-    "@bam.tech/react-native-image-resizer": ">=3.0.10",
-    "@react-native-camera-roll/camera-roll": ">=5.0.0",
-    "@react-native-clipboard/clipboard": ">=1.14.1",
-    "react-native": ">=0.73.6",
-    "react-native-audio-recorder-player": ">=3.6.4",
-    "react-native-blob-util": ">=0.19.9",
-    "react-native-document-picker": ">=9.0.1",
-    "react-native-haptic-feedback": ">=1.11.0",
-    "react-native-image-crop-picker": ">=0.33.2",
-    "react-native-share": ">=10.2.1",
-    "react-native-video": ">=5.2.1"
+    "@bam.tech/react-native-image-resizer": "^3.0.10",
+    "@react-native-camera-roll/camera-roll": "^5.0.0",
+    "@react-native-clipboard/clipboard": "^1.14.1",
+    "react-native": "^0.73.6",
```

Comment:
Perhaps you shouldn't change this one, as now package managers will warn in case someone uses 0.74.x

Reply:
maybe use `^0.73 || ^0.74 || ^0.75`

Reply:
We also allow versions below 0.73 like 0.72, 0.71 and SDK shouldn't break with this as well. So, I assume we shall mention it as well, right? But what's the limit here?

Reply:
May be support only 3 old versions?